src/ExperimentTable.js

```diff
@@ -0,0 +1,188 @@
+        _.flattenDeep(profile[tableHeader[this.state.searchedColumn].dataParam]).some(item=>item.toLowerCase().includes(searchQuery.toLowerCase())) :
+        profile[tableHeader[this.state.searchedColumn].dataParam].toString().toLowerCase().includes(searchQuery.toLowerCase())
+      )
+  }
+
+  setValue(property){
+    return e => {
+      this.setState({
+        [property]: e.target.value,
+        currentPage: 1
+      })
+    }
+  }
+
+  handleCheckbox(accession){
+    const checkedArray = this.state.checkedArray.slice()
+    checkedArray.includes(accession) ?
+      _.remove(checkedArray, function(n) {return n === accession})
+      : checkedArray.push(accession)
+    this.setState({checkedArray: checkedArray})
+  }
+
+  render() {
+    const {selectedSearch, selectedKingdom, checkedArray, selectedNumber, currentPage, searchedColumn, searchQuery, orderedColumn, ordering} = this.state
+    const {host, data, tableHeader, enableDownload} = this.props
+
+    const dataArray = selectedSearch ? this.sort(data).filter(data => data && Object.values(data).some(value => value.toString().toLowerCase().includes(selectedSearch.toLowerCase()))) :
+      this.filter(this.sort(data), tableHeader).filter(data => selectedKingdom ? data.kingdom === selectedKingdom : true)
+    const currentPageData = selectedNumber ? dataArray.slice(selectedNumber*(currentPage-1), selectedNumber*currentPage) : dataArray
+
+    return [
+      <div key={`tableHead`} className={`row expanded`}>
+        <div className={`large-2 medium-4 small-8 columns`}>
+          <label> Kingdom:
+            <select defaultValue={``} className={`kingdom`}
+              onChange={event => this.setState({selectedKingdom: event.target.value})}>
+              <option value={``} >All</option>
+              <option value={`animals`}>Plants</option>
+              <option value={`plants`}>Animals</option>
+              <option value={`fungi`}>Fungi</option>
+            </select>
+          </label>
+        </div>
+        <div className={`large-2 medium-4 small-8 columns`}>
+          <label>Entries per page:
+            <select defaultValue={10} onChange={this.setValue(`selectedNumber`)}>
+              <option value={10}>10</option>
+              <option value={25}>25</option>
+              <option value={50}>50</option>
+              <option value={100}>100</option>
+              <option value={data.length}>All</option>
+            </select>
+          </label>
+        </div>
+        <div className={`large-2 medium-4 small-8 columns`}>
+          <label>Search all columns:
+            <input type={`search`} placeholder={`Type here ...`}
+              onChange={this.setValue(`selectedSearch`)}/></label>
+        </div>
+      </div>,
+
+      <Table border key={`table`} style={{display:`grid`}}>
+        <Table.Head>
+          <Table.TextHeaderCell key={`index`} flexBasis={100} flexShrink={0} flexGrow={0}>Index</Table.TextHeaderCell>
+          {[
```

Comment:
You can tidy up the JSX a little bit here. You don't need to enclose `TableHeaderCell` in { } and you don't need the [ ].